CONTRIBUTING.md

```diff
@@ -2,6 +2,8 @@
 
 Thank you for your interest in contributing to SideStore! SideStore is a community driven project, and it's made possible by people like you.
 
+By contributing to this Project (SideStore), you agree to the Developer's Certificate of Origin found in [AGREEMENT.md](./AGREEMEND.md). Any contributions to this project after the addition of the Developer's Certificate of Origin are subject to its policy.
```

Comment:
Typo (links to nonexistent AGREEMEND.md)

Reply:
Cant currently fix it, you're welcome to (if able). It's on a branch not a fork.